webvtt.go

```diff
@@ -340,11 +345,10 @@ func unescapeWebVTT(i string) string {
 }
 
 // parseTextWebVTT parses the input line to fill the Line
-func parseTextWebVTT(i string) (o Line) {
+func parseTextWebVTT(i string, webVTTTagStack []WebVTTTag) (Line, []WebVTTTag) {
```

Comment:
Like for `srt`, could you replace with ` func parseTextWebVTT(i string, tags *[]WebVTTTag) (o Line)` instead?